lib/triggers/rollTrigger.js

```diff
@@ -38,15 +44,36 @@ RollTrigger.prototype._respond = function(roomId, userId, message) {
 
 	var that = this;
 	var params = that._stripCommand(message, that);
-	if(params) {
+	if(!params) {
+		return false;
+	}
+	var split = params.split('d');
+	if(parseInt(split[0]) === 1) {
+		try {
+			var rolled = dice.roll(params);
+			var result = rolled.result.toString();
+			if(parseInt(rolled.input.quantity > 1) {
+				result += " ("+JSON.stringify(rolled.rolled).replace(/,/g,", ")+")";
+			}
+			that._sendMessageAfterDelay(toId, result);
+		} catch(err) {
+			that._sendMessageAfterDelay(toId, "Try using syntax from www.github.com/troygoode/node-roll");
+		}
+	} else if(parseInt(split[0]) > this.options.maxDice || parseInt(split[1]) > this.options.maxSize) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/969592593/issues/source?bid=1980036&fileBranchId=2848073#l62)